packages/jest-snapshot/src/__tests__/__snapshots__/mock_serializer.test.js.snap

```diff
@@ -25,6 +25,7 @@ Object {
     "results": Array [
       Object {
         "isThrow": false,
+        "type": "return",
```

Comment:
having `type` here is still a breaking change, and we will probably get shit for it. I honestly don't care enough to change it (but I know the change is coming, so I'll just press `u` and go on with my day), but I'll happily point here from any issues opened 😛

Reply:
Don't we have precedent of putting minor snapshot updates in minor releases?

Reply:
I don't this we have without also getting complaints about semver. I might be mistaken though 🙂